approaching_emergency_vehicle_plugin/include/approaching_emergency_vehicle_plugin/approaching_emergency_vehicle_plugin_node.hpp

```diff
@@ -83,6 +85,7 @@ namespace approaching_emergency_vehicle_plugin
   struct UpcomingLaneChangeParameters{
     lanelet::ConstLanelet starting_lanelet; // The starting lanelet of the upcoming lane change
     lanelet::ConstLanelet ending_lanelet;   // The ending lanelet of the upcoming lane change
+    bool is_right_lane_change;              // Flag to indicate lane change direction; true if a right lane change
```

Comment:
If false, vehicle turn left or straight?

Reply:
If false, the vehicle lane changes to the left. Updated the comment in [51e9349](https://github.com/usdot-fhwa-stol/carma-platform/pull/2064/commits/51e93493ada73f639deff8da0a19ca2d1019e4af).